test/mongodb.test.js

```diff
@@ -805,8 +805,8 @@ describe('mongodb connector', function() {
     ) {
       Post.find(function(err, results) {
         events.should.eql([
-          'before execute insert',
-          'after execute insert',
+          'before execute insertOne',
+          'after execute insertOne',
```

Comment:
I think this is a breaking change from the point of loopback-connector-mongodb users.

An application installing `before execute` or `after execute` hook an relying on `ctx.req.command` value `insert` will stop working after upgrade.

IMO, this change should be released as semver-major.

/cc @virkt25  @dhmlau @raymondfeng

Reply:
Had a discussion with @raymondfeng and we came to the agreement that the minor release for this is fine as `mongodb` never bumped a major version for this release. 

Should there be an issue with this impacting users in a negative way we can look into reverting this commit and releasing a new minor version -- followed by a new major version containing this fix. Haven't seen any related issues thus far. 😄

Reply:
After reading up some more on connector hooks I changed my mind, discussed this further and have opened a PR to address this.